script/deps

```diff
@@ -0,0 +1,3 @@
+#/bin/bash
```

Comment:
I think what you want is `#!/bin/bash`

This should allow bash to be selected automatically from the CLI.

```bash
scripts/deps
```

instead of 
```bash
bash scripts/deps
```


See https://en.wikipedia.org/wiki/Shebang_(Unix)

Reply:
Great point! I also ran each script against `chmod 755` to get that execution bit in